packages/playground/src/components/account/account-edit/account-edit.tsx

```diff
@@ -9,6 +10,11 @@ import { UserChangeset } from "../../../types";
   shadow: true
 })
 export class AccountEdit {
+  @Element() el!: HTMLStencilElement;
+  @Prop() address: string = "";
+  @Prop() email: string = "";
```

Comment:
Why is there an email field

Reply:
@snario Shouldn't the account edit page allow editing of email? If not then I can remove this

Reply:
@snario @Alonski  Let's just leave this for now.